tests/ops/graph/test_config.py

```diff
@@ -74,6 +74,12 @@ def test_field_address_collection_address(self):
             "A", "B", "C", "D", "E"
         ).collection_address() == CollectionAddress("A", "B")
 
+    def test_field_address_from_string(self):
+        assert FieldAddress.from_string("A:B:C") == FieldAddress("A", "B", "C")
+        assert FieldAddress.from_string("A:B:C:D") == FieldAddress("A", "B", "C", "D")
+        with pytest.raises(FidesopsException):
+            assert FieldAddress.from_string("A:B")
```

Comment:
If there was ever a test failure it would be easier to debug if this was split up.

```suggestion
    @pytest.mark.parametrize("field_address, expected", [("A:B:C", FieldAddress("A", "B", "C")), ("A:B:C:D", FieldAddress("A", "B", "C", "D"))
    def test_field_address_from_string(self, field_address, expected):
        assert FieldAddress.from_string(field_address) == expected
        
    def test_field_address_from_string_error(self):
        with pytest.raises(FidesopsException):
            assert FieldAddress.from_string("A:B")
```

Reply:
Thanks, I've just removed!